### ISSUE TITLE
Linux aarch64 wheels

### ISSUE THREAD

**Issue by kuelumbus** [kuelumbus/rdkit-pypi#24]
Add aarch64 wheels

_Originally posted by @RobinFrcd in https://github.com/kuelumbus/rdkit-pypi/issues/22#issuecomment-1010909362_

**Comment by RobinFrcd** [comment]
Hello and thank you for the wheels!

However, it looks like some libs are missing:

```
In [1]: from rdkit.Chem.Draw import ReactionToImage
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-22522cb56040> in <module>
----> 1 from rdkit.Chem.Draw import ReactionToImage

/usr/local/lib/python3.7/site-packages/rdkit/Chem/Draw/__init__.py in <module>
     12 from importlib.util import find_spec
     13 
---> 14 from rdkit.Chem.Draw import rdMolDraw2D
     15 from rdkit.Chem.Draw.MolDrawing import MolDrawing, DrawingOptions
     16 from rdkit.Chem.Draw.rdMolDraw2D import *

ImportError: libGL.so.1: cannot open shared object file: No such file or directory
```

I tried installing rdkit on the same machine with conda, it works as expected.

I'm running on `Linux 5.11.0-1022-aws #23~20.04.1-Ubuntu SMP Mon Nov 15 14:04:48 UTC 2021 aarch64 GNU/Linux`.

**Comment by kuelumbus** [comment]
I don't think this is an issue with the wheel.  `libGL.so.1` is a system library that usually comes with your Linux distribution or is provided by the graphics driver (e.g., Nvidia). 

On Ubuntu you should be able to install it with `apt install libgl1`.  You might also need the package `libgl1-mesa-dev`.

Check out https://packages.ubuntu.com/search?suite=focal&arch=any&mode=filename&searchon=contents&keywords=libGL.so.1

**Comment by kuelumbus** [comment]
@RobinFrcd This should be fixed with `2021.9.5.1`. Please let me know if you still encounter problems.

**Comment by RobinFrcd** [comment]
Thank you so much, we'll have a go asap !

cc @plokkkk
